src/QueryBuilder/QueryBuilderModal/QueryBuilderModal.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const QueryBuilderModal = () => {
+  return (
+    <div>
+      Modal
```

Comment:
Is this text used by anything?

Reply:
No, it's just a placeholder for query builder modal which will be added in nearest feature